modules/persistent-storage-csi-olm-operator-install.adoc

```diff
@@ -27,7 +27,7 @@ To install the {FeatureName} CSI Driver Operator from the web console:
 +
 [IMPORTANT]
 ====
-Be sure to select the *AWS EFS CSI Driver Operator* and not the *AWS EFS Operator*. The *AWS EFS Operator* is a community Operator and is not supported by Red Hat.
+Be sure to select the *{FeatureName} CSI Driver Operator* and not the *{FeatureName} Operator*. The *{FeatureName} Operator* is a community Operator and is not supported by Red Hat.
```

Comment:
Here `AWS EFS CSI Driver Operator` and `AWS EFS Operator` is display name in operator-hub in web-console, I think we need to keep their name here.

Reply:
@duanwei33 Based on @codymant [comment](https://github.com/openshift/openshift-docs/pull/59650#issuecomment-1559974609), I believe we need to keep updated instance here. So, resolving this comment. Thank you!

Reply:
Hi @smunje1 I think that is not the case!
As I mentioned, they are display name in operator-hub in web-console and different ones, here I don't know what does {FeatureName}  stand for, but they should not be updated until they are changed from the operator-hub.
You can follow the doc to check.

Reply:
Sending you a snapshot from OCP operator-hub in web-console

Reply:
Here, {FeatureName} = Amazon EFS

Reply:
okay, so it should NOT be replaced. Could you please revoke it?

Reply:
I agree with @duanwei33 here. Operator Hub drivers and things in code like labels should not be updated.

Reply:
As per the above discussion, I am revoking this instance.

Reply:
Now, {FeatureName} = AWS EFS